src/main/kotlin/io/inventi/eventstore/projector/IdempotentEventHandler.kt

```diff
@@ -0,0 +1,175 @@
+package io.inventi.eventstore.projector
```

Comment:
I suggest naming package `eventhandler` or similar, since it (as @Zilv1nas mentioned) could be used for other purposes, e.g. sending mail